tests/async/test_async_auth.py

```diff
@@ -60,15 +61,52 @@ async def test_alogin_new_user(self):
         self.assertIsInstance(user, User)
         self.assertEqual(user.username, second_user.username)
 
-    async def test_alogin_without_user(self):
+    async def test_alogin_without_user_no_request_user(self):
+        request = HttpRequest()
+        request.session = await self.client.asession()
+        # RemovedInDjango61Warning: Fallback to request.user will be removed.
+        # This will raise AttributeError: 'NoneType' object has no attribute '_meta'
+        # when the fallback is removed.
+        with self.assertRaisesMessage(
+            AttributeError,
+            "'HttpRequest' object has no attribute 'auser'",
+        ):
+            await alogin(request, None)
+
+    async def test_alogin_without_user_anonymous_request(self):
+        async def auser():
+            return AnonymousUser()
+
+        request = HttpRequest()
+        request.user = AnonymousUser()
+        request.auser = auser
+        request.session = await self.client.asession()
+        # RemovedInDjango61Warning: Fallback to request.user will be removed.
+        # This will raise AttributeError: 'NoneType' object has no attribute '_meta'
+        # when the fallback is removed.
```

Comment:
If we plan to remove this test case, this comment is not required, otherwise should be:
```
        # RemovedInDjango61Warning: When the deprecation ends, replace with:
        # with self.assertRaisesMessage(
        #     AttributeError,
        #     "'NoneType' object has no attribute '_meta'",
        # ):
        #     await alogin(request, None)
```